snomed/com.b2international.snowowl.snomed.core.rest.tests/src/com/b2international/snowowl/snomed/core/rest/compare/SnomedMapTypeReferenceSetCompareTest.java

```diff
@@ -88,15 +89,15 @@ public void compareEqualSimpleMapTypeReferenceSets() {
 		ComponentURI baseURI = createURI(rf1Id);
 		ComponentURI compareURI = createURI(rf2Id);
 		
-		ConceptMapCompareResult result = CodeSystemRequests.prepareConceptMapCompare(baseURI, compareURI)
+		ConceptMapCompareResult result = CodeSystemRequests.prepareConceptMapCompare(baseURI, compareURI, ConceptMapCompareConfigurationProperties.DEFAULT_SELECTED_PROPERTIES)
```

Comment:
This can be the default value of the config field in the ConceptMap Compare API.